contentcuration/contentcuration/api.py

```diff
@@ -7,9 +7,21 @@
 
 from django.core.files import File as DjFile
 from django.db.models import Q
+from django.http import HttpResponse
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
 from kolibri.content.api import *
+from django.db import transaction
+import models
+
+def count_files(node):
```

Comment:
Oh, but this is counting `File` objects, for some reason? If it's for use in showing the number of resources within a topic, it should just count descendant content nodes.